nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
 
 snippet:rabbitmq-config-basic
 
-The transport needs to connect the the RabbitMQ broker. By default the transport will look for a connection string called `NServiceBus/Transport` in the `app.config`.
+
+## Specifying the connection string
```

Comment:
Can we split this into two headings? 
i..e Configuring via code as the first heading and then configuring via app.config? (since we prefer code first?)